### PR TITLE
Improve text of order overwrite confirmation dialog

### PR DESCRIPTION
**Pull request by windaishi**
### 1. Why is this change necessary?
The caption of the dialog, that is shown in case of a possible data loss when saving the order, is not explaining what has happened.

image

### 2. What does this change do, exactly?
Improves the caption of the yes-no dialog. It now explains what actually has happened and why the saving of the order has been aborted.

image

### 3. Describe each step to reproduce the issue or behaviour.
Open the same order in two different backend sessions.
User 1: Save an order.
User 2: Save the same order again.

### 4. Please link to the relevant issues (if any).
N/A

### 5. Which documentation changes (if any) need to be made because of this PR?
N/A

### 6. Checklist

- [x] I have squashed any insignificant commits
- [x] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: snippets/backend/order/main.ini [hunk 1]
```diff
@@ -89,7 +89,7 @@ convertOrder/message = "Do you want to convert this order to a regular order?"
 convertOrder/title = "Convert order?"
 convertOrderSuccess/tile = "Order has been converted"
 overwriteOrder/title = "Overwrite most recent changes"
-overwriteOrder/message = "Do you really want to overwrite the latest changes?"
+overwriteOrder/message = "The order has been changed by another user in the meantime. To prevent overwriting these changes, saving the order was aborted. To show this changes, please close the order and re-open it.<br /><br /><b>Do you want to overwrite the latest changes?</b>"
```
Comment (fixpunkt) [inline snippets/backend/order/main.ini]: > **these** changes

File: themes/Backend/ExtJs/backend/order/controller/list.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ Ext.define('Shopware.apps.Order.controller.List', {
         failureTitle: '{s name=message/save/error_title}Error{/s}',
         overwriteOrder: {
             title: '{s name=overwrite_order/title}Overwrite most recent changes{/s}',
-            message: '{s name=overwrite_order/message}Do you really want to overwrite the latest changes?{/s}',
+            message: '{s name=overwrite_order/message}The order has been changed by another user in the meantime. To prevent overwriting these changes, saving the order was aborted. To show this changes, please close the order and re-open it.<br /><br /><b>Do you want to overwrite the latest changes?</b>{/s}',
```
Comment (fixpunkt) [inline themes/Backend/ExtJs/backend/order/controller/list.js]: > **these** changes

**Review by fixpunkt — CHANGES_REQUESTED** [review]

**Comment by fixpunkt** [comment]
This is quite important for us because overwriting other changes made to an order by Shopware WMS will cause severe issues, up to and including corruption of stock records. So we really need the user to understand the consequences of clicking "Yes" here.

**Comment by shyim** [comment]
thanks for your contribution @windaishi  +1 :tada: :blue_heart:

**Pull request closed without merging**